command/server.go

```diff
@@ -1019,6 +1040,7 @@ func (c *ServerCommand) Run(args []string) int {
 		RedirectAddr:              config.Storage.RedirectAddr,
 		StorageType:               config.Storage.Type,
 		HAPhysical:                nil,
+		ConfigServiceDiscovery:    configSD,
```

Comment:
I think it's confusing to prefix this member variable (and the `configSD` variable) with `config`. Being part of the CoreConfig struct should be indication enough of where it came from.

Reply:
fixed